### PR TITLE
Sw bundle with rollup

### PR DESCRIPTION
**Pull request by shuowu**
This PR uses `rollup` instead `babel` to build/bundle the SDK, three bundles are generated:

- CommonJS: linked from pkg.main
- ES module: linked from pkg.module
- UMD: minified bundle can be used in both node and browser env, can be used in <script>

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -5,19 +5,26 @@ const chalk = require('chalk');
 const fs = require('fs');
 
 const NPM_DIR = `dist`;
-const BABEL_CMD = `babel src -d ${NPM_DIR}/src -s`;
+const BABEL_CMD = 'yarn bundle';
```
Comment (swiftone) [inline build.js:8]: This doesn't look like it should be BABEL_CMD

File: src/LoginCallback.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017-2020Present, Okta, Inc. and/or its affiliates. All rights reserved.
```
Comment (swiftone) [inline src/LoginCallback.js:2]: A space is missing somewhere...

File: util/maintain-banners.js [hunk 3]
```diff
@@ -3,8 +3,7 @@ const globby = require('globby');
 const path = require('path');
 
 const bannerSourcePath = path.join(__dirname, 'license-template.txt');
-const files = globby.sync(path.join(__dirname, '..','packages/*/{index.js,index.ts,lib.js,src/**/*.{js,ts},test/integration-test/*.{js,ts},test/e2e/**/*.{js,ts}}'));
-// console.log(files);
+const files = globby.sync(path.join(__dirname, '..','{src/**/*.js,test/**/*.js,dist/bundles/*.js}'));
```
Comment (swiftone) [inline util/maintain-banners.js:6]: There's a logic here, perhaps we can explain it in a comment?  All JS files of ours?

File: src/LoginCallback.js (hunk 2 above)
Comment (shuowu) [reply src/LoginCallback.js:2]: we can either keep 2020 or present, checked other repos, looks like `Present` is the consistent way.

**Review by swiftone — APPROVED** [review]

**Comment by shuowu** [comment]
How the dist folder looks like now:
Screen Shot 2020-11-13 at 3 39 49 PM

**Review by aarongranick-okta — APPROVED** [review]
It looks there is a change which affects devs (us), which is that the jest tests are run against the transpiled commonjs bundle. I think this means we would want to run `yarn dev` in a 2nd terminal while actively developing tests. Maybe a mention of this could be added at the end of the README in the "Development" section.

**Pull request merged**